src/Dropdown/Dropdown.tsx

```diff
@@ -65,10 +90,23 @@ const Dropdown: React.FC<DropdownProps> = ({
                 onClick={onClick}
                 disabled={disabled}
             >
-                <span>{selected?.label}</span>
+                <span>
+                    {items.findIndex(e => e.value === selected.value) === -1
+                        ? ''
+                        : selected.label}
+                </span>
                 <img className={styles.image} src={chevron} alt="" />
             </button>
             <div
+                style={
+                    noItemsBeforeScroll > 0
+                        ? { maxHeight: `${noItemsBeforeScroll * 24}px` }
+                        : {}
+                } // 24px from Dropdown.module.scss .item height
```

Comment:
Instead of hard coding and explaining in a comment the `24`, can you instead export the value from the SCSS (probably using an [`:export` pseudo-selector](https://github.com/css-modules/icss#export)) and use that in here?